unit_tests/rating_curve_comparison_test.py

```diff
@@ -21,33 +22,9 @@ def setUpClass(self):
             self.params = json.load(params_file)
 
 # Test Cases
-
-    def test_generate_rating_curve_metrics_01010004_success(self):
-
-        '''
-        We are testing whether a .png file was created for a FIM-USGS rating curve comparison, 
-        for HUC 0101004 using the `generate_rating_curve_metrics` function.
-        The 5th index (parameter) for each HUC in `rating_curve_comparison_params.json` specifies
-        the FIM-USGS rating curve comparison .png filepath.
-        '''
-        params = self.params["valid_data"].copy()  #update "valid_data" value if you need to (aka.. more than one node)
-
-        _indiv_huc_params = params["01010004"]
-
-        # To setup the test, lets start by deleted the expected output file to ensure
-        # that it is regenerated.
-        if os.path.exists(_indiv_huc_params[5]):
-            os.remove(_indiv_huc_params[5])
-        
-        # Test that the file was deleted
-        assert os.path.exists(_indiv_huc_params[5]) == False
-
-        generate_rating_curve_metrics(_indiv_huc_params)
-
-        # Test that the file was created by generate_rating_curve_metrics       
-        assert os.path.exists(_indiv_huc_params[5]) == True
-        
-
+     
+    @pytest.mark.skip(reason="rating_curve has invalid pathing based on files "\
```

Comment:
For what its worth, I was able to successfully run this test on `dev1` from this branch, as well as the two other tests in this file which were deleted.

Reply:
Yes.. and it would run on dev1, but it doesn't run on  USC2, UCS3 or EFS currently. Unless, we copy over the entire fim_unit_test_data_do_not_remove folder by hand to those enviros. According to the README.md, I can just run fim_pipeline.sh (as mentioned in the doc) and the output folder will be correct to run all test cases. 

However, somewhere in the past, Carson added a bunch of files that naturally don't exist and are never added to the output folder. So..  we not replicate getting those extra files that rating curve unit test needs to be run. So.. I disabled it for now, until we figure out how we will  handle this. It might be that we need to run a full alpha test on the HUCs created in the fim_unit_test_data_do_not_remove folder which will create folders in other places and we can point the json file to those folders (like test_cases).  I am not even sure that will work as I don't recognize those files that Carson added in there by hand at all. The whole unit test for rating is not part of the flow but a one off file and might not be a reasonable unit test in the first place. Research required at a later time.